gating/update_dependencies/run

```diff
@@ -52,27 +100,23 @@ release_data_file="${WORKSPACE}/rc-release-data.yml"
 # new method
 if git cat-file -e ${new_file_to_fetch} 2>/dev/null; then
   git show ${new_file_to_fetch} > ${release_data_file}
-  export RC_BRANCH_VERSION=$(${GATING_PATH}/../../scripts/get-rpc_release.py ${release_data_file})
+  export RC_BRANCH_VERSION=$(${BASE_DIR}/scripts/get-rpc_release.py -f ${release_data_file})
 
 # old method
 elif git cat-file -e ${old_file_to_fetch} 2>/dev/null; then
   git show ${old_file_to_fetch} > ${release_data_file}
   export RC_BRANCH_VERSION=$(awk '/^rpc_release/{print $2}' ${release_data_file} | tr -d '"')
 
 else
-  echo "RC branch ${rc_branch} not found, skipping rpc_release version bump.
-If there is no RC branch then the mainline branch is considered unreleased
-and therefore the rpc_release value is left alone. It is still important
-for the dependencies to be updated regularly though, so that part continues
```

Comment:
If you move `for` to the previous line it will pass the test.

Reply:
Great, thanks!